Sources/EmbraceConfigInternal/EmbraceConfig.swift

```diff
@@ -99,47 +53,38 @@ public class EmbraceConfig {
         }
 
         update()
+        lastUpdateTime = Date().timeIntervalSince1970
         return true
     }
 
     public func update() {
-        guard updating == false else {
-            return
-        }
-
-        updating = true
-
-        fetcher.fetch { [weak self] payload in
-            if let payload = payload {
-                let previousPayload = self?.payload
-
-                self?.payload = payload
-
-                if previousPayload != payload {
-                    self?.notificationCenter.post(name: .embraceConfigUpdated, object: self)
```

Comment:
This functionality seems to be lost now.
Was it replaced with anything else? I know some parts of the SDK need this notification.

Reply:
Yup, need to add this back in :+1: